data/bin/normalize-county-production.js

```diff
@@ -28,10 +24,11 @@ var async = require('async');
 var thru = require('through2').obj;
 
 var statesByAbbr;
-var yearField = 'Calendar Year';
-var fipsField = options['fips-field'];
+var yearField = 'Year';
 var stateField = options['state-field'];
-var volumeField = options['volume-field'];
+var volumeField = 'Production Volume';
+
+const WITHHELD = 'Withheld';
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).